src/ProjectService.Broker/DisactivateUserConsumer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using LT.DigitalOffice.Models.Broker.Common;
+using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using MassTransit;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ProjectService.Broker
+{
+    public class DisactivateUserConsumer : IConsumer<IDisactivateUserRequest>
+    {
+        private readonly IUserRepository _repository;
+
+        public DisactivateUserConsumer(
+            IUserRepository repository)
+        {
+            _repository = repository;
+        }
+
+        public Task Consume(ConsumeContext<IDisactivateUserRequest> context)
+        {
+            _repository.Remove(context.Message.UserId);
+
+            return Task.FromResult(0);
```

Comment:
Need return result executed operation

Reply:
We don't have to return anything here, becouse it is not for request client, it is for publish

Reply:
if use simple return